ExpressApp/app.js

```diff
@@ -1,7 +1,12 @@
 const express = require("express");
-const { getApiHealth, getApiEndpoints } = require("./controllers/api-healthcheck-controller");
+const {
+  getApiHealth,
+  getApiEndpoints,
+} = require("./controllers/api-healthcheck-controller");
 const { handleNotFound } = require("./api-errors");
 const { getApiTopics } = require("./controllers/api-topics-controller");
+const { getArticleById } = require("./controllers/api-articles-controller");
+const { invalidInput } = require("./sql-db-errors");
 
 const app = express();
 
```

Comment:
can you please remove the express.json(), while you may need it in the future currently you do not.